client/blocks/reader-subscription-list-item/docs/example.jsx

```diff
@@ -0,0 +1,69 @@
+/**
+ * External dependencies
+ */
+import React, { PureComponent } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import ReaderSubscriptionListItem from '../';
+import Card from 'components/card';
+import { getCurrentUser } from 'state/current-user/selectors';
+
+
+const items = [
+	{
+		siteAuthor: {
+			avatar_URL: 'https://2.gravatar.com/avatar/83cb16245c3d8ac5f625ebf91a7827e7?d=identicon&r=G&s=1600A',
+			first_name: 'Allison',
+			last_name: undefined,
+		},
+		siteUrl: 'http://afarmgirlslife.wordpress.com',
+		siteTitle: 'A Farm Girls Life',
+		siteExcerpt: 'Photography, Crafty Things, and Life on the Farm',
+		lastUpdated: 'x days ago',
+		feedId: '21587482',
+	},
+	{
+		siteAuthor: {
+			avatar_URL: 'https://2.gravatar.com/avatar/bd02f74face048cc62c4eda28cea9937?d=mm&r=G&s=96',
+			first_name: undefined,
+			last_name: undefined,
+		},
+		siteUrl: 'https://fourthgenerationfarmgirl.com/',
+		siteTitle: 'fourth generation farmgirl',
+		siteExcerpt: 'wool and wine to tractors and travel',
+		lastUpdated: 'x days ago',
```

Comment:
Is it better for the list item component to receive the date, and figure this out for itself? It'll need to be translated.